mage/algorithms/dynamic-graph-analytics/betweenness-centrality-online-algorithm.md

```diff
@@ -0,0 +1,88 @@
+---
+id: betweenness-centrality-online-algorithm
+title: Dynamic Betweenness Centrality
+sidebar_label: Dynamic Betweenness Centrality
+---
+
+## Description
+
+Betweenness centrality is among the most common metrics in graph analytics owing
+to its utility in identifying critical vertices of graphs. It is one of the
+tools in _centrality analysis_, a set of techniques for measuring the importance
+of nodes in networks.
+
+The notion of [Betweenness
+centrality](https://en.wikipedia.org/wiki/Betweenness_centrality) is based on
+shortest paths: the shortest path between two nodes is the one consisting of the
+fewest edges, or in case of weighted graphs, the one with the smallest total
+edge weight. A node’s betweenness centrality is defined as the share of all
+shortest paths in the graph that run through it.
+
+MAGE includes a _fully dynamic_ betweenness centrality computation tool that
+implements the
+[iCentral](https://repository.kaust.edu.sa/bitstream/handle/10754/625935/08070346.pdf)
+[^1] algorithm. iCentral saves up on computation in two ways: it singles out the
```

Comment:
Write the reference below and name the author

Reply:
I’ve added the authors’ names. The reference is at the bottom of **Abstract**, is that OK?

Reply:
Yes!